### DISCUSSION TITLE
React Native

### DISCUSSION THREAD

**Discussion by adrianotadao** [rainbow-me/rainbowkit#1669] [Q&A] [answered]
hey there, are you using Rainbowkit on react native somehow? If so, could you share it with us, please?

as far as I can tell, there is no good/pretty option for it, am I wrong?

Thank you very much.

**Comment by 0xmkh** [comment, marked as answer]
@adrianotadao We do not support RainbowKit in react native. Is this something that you would like us to look at ?

**Reply by adrianotadao** [reply]
yes, please. 

Rainbowkit is the most beautiful library for wallet connection and that would be great if it also supports React native
